src/pages/overview-page/catalog-item/catalog-item.component.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../../node_modules/designsystemet/fdk-designsystem-bootstrap4/scss/colors.scss';
```

Comment:
Fjerne hele denne filen. Se kommentar for catalog-item under.
Tror heller ikke det er nødvendig med all styling her, f.eks. det som gjelder hover kan du sikkert utelate for å gjøre det enklere nå. Designet kommer vel senere?

Reply:
48f3e62

Ja, i første omgang tar vi bare inn one som ligner på registrering-portal. Så ser vi på skikkelig design på et senere tidspunkt